server/src/main/java/com/warehousepro/service/UserService.java

```diff
@@ -23,6 +28,8 @@ public class UserService {
   UserRepository userRepository;
   UserMapper userMapper;
   final PasswordEncoder passwordEncoder;
+  RoleRepository roleRepository;
```

Comment:
Wrong service, `RoleRepository` only import in `RoleService`, `PermissionRepository` only import in `PermissisonService`